sources/web/datalab/polymer/components/datalab-files/datalab-files.html

```diff
@@ -49,6 +49,10 @@
         width: 20px;
         padding-right: 5px;
       }
+      .toolbar-button[disabled] {
+        color: var(--disabled-fg-color);
+        background-color: var(--secondary-bg-color);
```

Comment:
That's because by default a button's disabled state applies a different background color, I'm overriding that to keep it consistent with the toolbar's background color.